### DISCUSSION TITLE
Fetching data only once

### DISCUSSION THREAD

**Discussion by davidstackio** [vuejs/vuefire#1321] [Questions and Help] [answered]
I see there is a [`once` option](https://github.com/vuejs/vuefire/blob/97a53eac8cca3ee0a00b61c154d8d8e605e311de/src/shared.ts#L254) for fetching data only once (instead of subscribing to realtime updates), but it is labeled as experimental in the code. Is this still the case?

**Comment by posva** [comment, marked as answer]
It's not experimental anymore. It's used internally for SSR as well.
